### ISSUE TITLE
[BUG] Undefined instead of nickname in +hstats

### ISSUE THREAD

**Issue by Syjalo** [Hypixel-Translators/hypixel-translators-bot#230]
**Describe the bug**
If you execute `+hstats` after executing `+hverify`, then you will have `undefined` instead of the nickname.

**To Reproduce**
1. Execute '+hverify \<nickname>'
2. Execute '+hstats'
3. See error

**Expected behavior**
I should get my nickname.

**Screenshots**
![upload_2021_02_10_07_40_55_477](https://user-images.githubusercontent.com/62868468/107457119-510ab780-6b73-11eb-9538-c24540c3f792.jpg)

**Solving**
Use `username` instead of `args[0]`.
[Go to the file](https://github.com/Hypixel-Translators/hypixel-translators-bot/blob/97ab968e8f22706694ad12e205c709befe5dca3f/commands/Utility/hypixelstats.js#L91)

**Comment by ImRodry** [comment]
Thanks for the report! This should be an easy fix and I’ll look into it today.

**Comment by ImRodry** [comment]
> Use username instead of args[0].

I missed this comment but the reason we don't use this is that once a player changes their name and doesn't log on to Hypixel, the command will only work with the new name and username will be the last name they used to log on to the server. If we used username by default it could cause issues with these types of users. The solution I found was to add username as an alternative for when args[0] is undefined.
